api/index.coffee

```diff
@@ -14,7 +15,22 @@ app.use cors()
 app.use helpers
 app.use bodyParser.json limit:'5mb', extended: true
 app.use bodyParser.urlencoded limit:'5mb', extended: true
-app.use morgan(if NODE_ENV is 'development' then 'dev' else 'combined')
+
+app.use morgan if NODE_ENV is 'production' then 'dev' else (tokens, req, res) ->
```

Comment:
@kanaabe Nice pattern!  Can we extract this somehow to include across other Express apps using the Morgan logger?